content/release-notes/2022-09-28.md

```diff
@@ -0,0 +1,21 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Console: Added `read`, `update`, `delete` project access scopes for OAuth applications.
+- Console: Added possibility to select access scopes and projects on the OAuth consent page.
+- Console: Added `protected: boolean` flag into roles public API response.
+- Console: Changed region and platform ids from serial numbers to strings in the public API.
+- Console: Changed operations id from serial number to UUID type in the public API.
+- Control Plane: Added internal mechanism for projects relocation between pageservers.
+- Control Plane: Stopped sending metrics to Grafana Cloud.
+- Control Plane: Removed usage of the obsolete `zenith_ctl` binary.
+- Control Plane: Split monolithic project object into project, branches and endpoints in the internal database schema.
+
+### Bug fixes
+
+- UI: Fixed failure after receiving the empty response in the SQL Editor.
+- UI: Fixed an issue when switching between schemas in Tables UI did not reset the table name.
+- Control Plane: Fixed passwordless proxy connection to just created project.
```

Comment:
```suggestion
- Control Plane: Fixed the passwordless proxy connection for the most recently created project.
```

Reply:
Ho @ololobus How about:
"Fixed an issue that occurred when creating a new project using a passwordless proxy connection.

Reply:
Sounds good to me! Reworded a bit, though